src/PharoDocComment-Tests/PharoDocCommentTest.class.st

```diff
@@ -96,8 +99,8 @@ PharoDocCommentTest >> testExpressionZero [
 	| node |
 	node := thisContext method ast pharoDocCommentNodes first.
 	self assert: node expression isFaulty.
-	self should: [ node expression evaluate ] raise: RuntimeSyntaxError.
-	self should: [ (CommentTestCase for: node) testIt ] raise: RuntimeSyntaxError
+	self should: [ node expression evaluate ] raise: Error.
```

Comment:
Error is too generic

Reply:
I will add a dedicated Exception in a next PR